contracts/tg4-mixer/benches/main.rs

```diff
@@ -48,12 +48,12 @@ fn main() {
 
     println!();
     for (poe_fn_name, poe_fn, result, gas) in [
-        ("GeometricMean", GeometricMean {}, 22360, 5721600000),
+        ("GeometricMean", GeometricMean {}, 22360, 5729550000),
```

Comment:
So the gas usage increased, not much though.
Would be better to keep here some approximation instead of fixed full value?